eng/pipelines/libraries/build-job.yml

```diff
@@ -92,6 +92,10 @@ jobs:
             parameters:
               installDotnet: true
 
+        # Build CoreLib
+        - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -subsetCategory coreclr -subset corelib $(crossArg) -arch $(archType) -c ${{ parameters.liveRuntimeBuildConfig }} $(officialBuildIdArg) -ci
```

Comment:
I think we should be able to combine this into one step.

Reply:
"all" as subset didnt seem to work, but this appears to work: 

build -subsetCategory coreclr-libraries -subset corelib-depprojs-ref-native-src -runtimeConfiguration Release

Reply:
also it appears @ViktorHofer just merged some refactor of build scripts. Does it still support building multiple products? Need something like this:
"build -subset clr.corelib-Libs -runtimeConfiguration Release"

Reply:
Yeah it would look like: `build -subset clr.corelib-libs -runtimeConfiguration ${{ parameters.liveRuntimeConfig }} .. other args`

Reply:
doesnt appear to do the right thing for me, appears to only build clr.corelib, not libs.

Reply:
never mind, appears to be a '**+**' instead of '-' now :)

Reply:
Right, sorry about that, muscle memory 😄 .